scripts/setup_btx.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+usage()
+{
+cat << EOF
+$(basename "$0"):
+    Script to prepare btx directories and eLog jobs for an experiment.
+    OPTIONS:
+      -a | --account
+         Optionally specify an account to run with.
+      -e | --experiment
+         Experiment to setup btx for.
+      -h | --help
+         Display this help message.
+      -n | --ncores
+         Number of cores to use for btx jobs.
+      -q | --queue
+         Which S3DF queue to run on.
+      -r | --reservation
+         Optionally specifiy a reservation to run on.
+      -v | --verbose
+         Optionally enable verbose logging.
+      -w | --workflow
+         Specify which workflow to setup.
+EOF
+}
+
+ARGS=()
+while [[ $# -gt 0 ]]
+do
+    arg="$1"
+
+    case $arg in
+        -a|--account)
+            ACCOUNT="$2"
+            shift
+            shift
+            ;;
+        -e|--experiment)
+            EXP="$2"
+            shift
+            shift
+            ;;
+        -h|--help)
+            usage
+            exit
+            ;;
+        -n|--ncores)
+            CORES="$2"
+            shift
+            shift
+            ;;
+        -q|--queue)
+            QUEUE="$2"
+            shift
+            shift
+            ;;
+        -r|--reservation)
+            RESERVATION="$2"
+            shift
+            shift
+            ;;
+        -v|--verbose)
+            VERBOSE=1
+            shift
+            shift
+            ;;
+        -w|--workflow)
+            WORKFLOW="$2"
+            shift
+            shift
+            ;;
+        *)
+            ARGS+=("$1")
+            shift
+            shift
+            ;;
+    esac
+done
+set -- "${ARGS[@]}"
+
+if [[ -z ${EXP} ]]; then
+    echo "You must provide an experiment name!"
```

Comment:
We could also use the default provided by https://github.com/pcdshub/engineering_tools/blob/master/scripts/get_info

Reply:
I anticipate this will hopefully be used before the beamtime, so I think it may be safer to force the experiment is specified than taking the active one.

Does this script need the hutch to pull the current configuration as well?

Reply:
Good point, let's not add that.

I'm not super familiar with this script yet.